js/viz/bar_gauge.d.ts

```diff
@@ -174,6 +174,13 @@ export interface dxBarGaugeOptions extends BaseWidgetOptions<dxBarGauge> {
        */
       startAngle?: number;
     };
+    /**
+     * @docid
+     * @default undefined
+     * @type_function_return string|SVGElement|jQuery
+     * @public
+     */
+    centerTemplate?: template | ((component: dxBarGauge, element: SVGGElement) => string | UserDefinedElement<SVGElement>);
```

Comment:
@OlegKipchatov, please remind - do we need something special here for templates? `@type template` or like?